index.bs

```diff
@@ -1043,17 +1072,19 @@ partial interface HTMLIFrameElement {
     <div class="algorithm"
     data-algorithm="define-inherited-policy-in-container">
     Given a [=policy-controlled feature|feature=] (|feature|), null or a <a>navigable container</a>
-    (|container|), and an <a for="Document">origin</a> for a {{Document}} in
-    that container (|origin|), this algorithm returns the [=inherited policy for
-    a feature|inherited policy value=] for |feature|.
+    (|container|), an <a for="Document">origin</a> for a {{Document}} in
```

Comment:
The inherited policy for a child document has always been based on the enforced policy in the parent; not the report-only policy of the parent. Report-only policies are local, essentially.

I think what you've done here is correct,  since the default is false, and it's only overridden by the algorithm which tests for potential violations, but I just want to make sure that there's no intention to have report-only policies themselves ever be inherited by child documents.